src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/generate.cmd

```diff
@@ -13,7 +13,7 @@ if  "%1" == "" (
     set specFile="%1"
 )
 
-set repoRoot=%~dp0..\..\..
+set repoRoot=%~dp0..\..\..\..\..
```

Comment:
You need to update this file so that the specFile points to a specific commit.  In this case: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/6e2ee40c8e74887d2266d750ce403c81ef2402a6/keyvault/2016-10-01/swagger/keyvault.json

This protects you from upcoming changes to the specs repo.